src/prefect/agent/local/agent.py

```diff
@@ -55,6 +55,8 @@ class LocalAgent(Agent):
         - hostname_label (boolean, optional): a boolean specifying whether this agent should
             auto-label itself with the hostname of the machine it is running on.  Useful for
             flows which are stored on the local filesystem.
+        - storage_labels (boolean, optional): a boolean specifying whether this agent should
+            auto-label itself with all of the available storage options labels.
```

Comment:
minor nit, does it make sense to have "available" in this description (and in similar below)? It seems to imply that we are doing more than attaching all of the possible storage option labels.

Reply:
Works for me! Fixed in 5bafac89f2efb74a5894ec79905c1a7a72354dba